auth/ipa_2fa.adoc

```diff
@@ -0,0 +1,86 @@
+[[two-factor-authentication]]
+== 2-Factor Authentication
+
+Enabling 2-Factor Authentication on the Appliance is currently supported
+with External Authentication to IPA.
+
+In this guide we will cover how to manually configure a 7.1 Appliance's
+external authentication to work with 2-Factor Authentication with IPA.
+This provides IPA Users access to the Appliance Administrative UI and the REST API
+using their IPA Password followed by a One-Time-Password.
+
+1.  <<requirements, Requirements>>
+2.  <<configuring-admin-ui, Configuring the Administrative UI>>
+3.  <<configuring-external-auth, Configuring External Authentication>>
+4.  <<enabling-2fa, Enabling 2-Factor Authentication>>
+5.  <<using-2fa, Using 2-Factor Authentication>>
+
+[[requirements]]
+== Requirements
+
+The following is needed in order to enable 2-Factor-Authentication to the Appliance:
+
+* A CentOS/RHEL 7.1 based Appliance
+* IPA Server based on FreeIPA 4.1.0 or later
+
+[[configure-admin-ui]]
+== Configure Administrative UI 
+
+Login as admin, then in _Configure->Configuration->Authentication_
+
+* Set mode to External (httpd)
+* Check: _Get User Groups from External Authentication (httpd)_
+* Do Not Check: _Enable Single Signon_
+** *Note*: Kerberos SSO with OTP is not supported in the current release.
```

Comment:
In the current release of IPA.   I'll update.

On Wed, Sep 9, 2015 at 9:40 AM, Marianne Feifer <notifications@github.com>
wrote:

> In auth/ipa_2fa.adoc
> <https://github.com/ManageIQ/manageiq_docs/pull/5#discussion_r39042582>:
>
> > +== Requirements
> > +
> > +The following is needed in order to enable 2-Factor-Authentication to the Appliance:
> > +
> > +* A CentOS/RHEL 7.1 based Appliance
> > +* IPA Server based on FreeIPA 4.1.0 or later
> > +
> > +[[configure-admin-ui]]
> > +== Configure Administrative UI
> > +
> > +Login as admin, then in _Configure->Configuration->Authentication_
> > +
> > +* Set mode to External (httpd)
> > +* Check: _Get User Groups from External Authentication (httpd)_
> > +* Do Not Check: _Enable Single Signon_
> > +** *Note*: Kerberos SSO with OTP is not supported in the current release.
>
> What do you mean by "current release"? Do you mean not in Botvinnik? So it
> will be in Capoblanco?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/manageiq_docs/pull/5/files#r39042582>.
>



-- 
*Alberto Bellotti*
abellott@redhat.com